base-server/index.js

```diff
@@ -409,7 +471,7 @@ export class BaseServer {
     return [undoAction, undoMeta]
   }
 
-  channel(pattern, callbacks) {
+  channel(pattern, callbacks, options) {
```

Comment:
```suggestion
  channel(pattern, callbacks, options = {}) {
```

It will allow to replace `options?.queue || 'main'` to `options.queue || 'main'`